README.md

```diff
@@ -119,3 +117,53 @@ This example also comes with a robot test file:
 cd renode_1.12.0_portable
 test.sh --variable ELF:${PATH_TO_BINARY} ${PATH_TO_REPOSITORY_ROOT}/tests/simple_embedding.robot
 ```
+
+### Custom memory configuration
+Additionally a custom memory configuration exists for the STM32F4 board.
+This configuration increases the size of the ROM and RAM to 256M each.
+
+To use this custom memory configuration the target needs to be built with the appropriate linker script that accomodate the extended memory.
+The build process is largely the same as before, only the linker scripts specified in ${PATH_TO_LINKER_SCRIPT} have been amended.
+
+```shell
+mkdir build
+cd build
+
+# Set the path to the GNU Arm Embedded Toolchain, e.g.
+# export PATH_TO_ARM_TOOLCHAIN="/usr/local/gcc-arm-none-eabi-10-2020-q4-major"
+
+# To build with CMSIS
+# export CUSTOM_ARM_LINKER_FLAGS="-lnosys"
+# export PATH_TO_LINKER_SCRIPT="`realpath ../build_tools/stm32f4/stm32f407-cmsis-custom.ld`"
+
+# To build with libopencm3
+# export CUSTOM_ARM_LINKER_FLAGS="-nostartfiles"
+# export PATH_TO_LINKER_SCRIPT="`realpath ../build_tools/stm32f4/stm32f4-discovery-libopencm3-custom.ld`"
+
+
+# export PATH_TO_IREE_HOST_BINARY_ROOT="`realpath ../build-iree-host-install`"
+
+cmake -GNinja \
+#     -DBUILD_WITH_CMSIS=ON \
+#     -DBUILD_WITH_LIBOPENCM3=ON \
+      -DCMAKE_TOOLCHAIN_FILE="`realpath ../build_tools/cmake/arm.toolchain.cmake`" \
+      -DARM_TOOLCHAIN_ROOT="${PATH_TO_ARM_TOOLCHAIN}" \
+      -DARM_CPU="armv7e-m" \
+      -DIREE_HAL_DRIVERS_TO_BUILD="VMVX_Sync" \
+      -DIREE_HOST_BINARY_ROOT="${PATH_TO_IREE_HOST_BINARY_ROOT}" \
+      -DCUSTOM_ARM_LINKER_FLAGS="${CUSTOM_ARM_LINKER_FLAGS}" \
+      -DLINKER_SCRIPT="${PATH_TO_LINKER_SCRIPT}" \
+      ..
+cmake --build . --target sample_vmvx_sync
+```
```

Comment:
With only `PATH_TO_LINKER_SCRIPT` changed, I don't see a need to replicate the whole instructions. This only makes it harder to keep our docs up-to-date. I would strongly prefer if we can drop the duplication.